commands/shutdown.md

```diff
@@ -41,11 +46,28 @@ instead. This happens when:
 
 * The user just turned on AOF, and the server triggered the first AOF rewrite in order to create the initial AOF file. In this context, stopping will result in losing the dataset at all: once restarted, the server will potentially have AOF enabled without having any AOF file at all.
 * A replica with AOF enabled, reconnected with its master, performed a full resynchronization, and restarted the AOF file, triggering the initial AOF creation process. In this case not completing the AOF rewrite is dangerous because the latest dataset received from the master would be lost. The new master can actually be even a different instance (if the **REPLICAOF** or **SLAVEOF** command was used in order to reconfigure the replica), so it is important to finish the AOF rewrite and start with the correct data set representing the data set in memory when the server was terminated.
+* The **FORCE** modifier is *not* specified. If **FORCE** is specified, the server exits anyway.
+
+There are conditions when we want just to terminate a Redis instance ASAP, regardless of what its content is.
+In such a case, the command **SHUTDOWN NOW NOSAVE FORCE** can be used.
+In versions before 7.0, where the **NOW** and **FORCE** flags are not available, the right combination of commands is to send a **CONFIG appendonly no** followed by a **SHUTDOWN NOSAVE**.
+The first command will turn off the AOF if needed, and will terminate the AOF rewriting child if there is one active.
+The second command will not have any problem to execute since the AOF is no longer enabled.
 
-There are conditions when we want just to terminate a Redis instance ASAP, regardless of what its content is. In such a case, the right combination of commands is to send a **CONFIG appendonly no** followed by a **SHUTDOWN NOSAVE**. The first command will turn off the AOF if needed, and will terminate the AOF rewriting child if there is one active. The second command will not have any problem to execute since the AOF is no longer enabled.
+## Minimize the risk of data loss
+
+Since Redis 7.0, the server waits for lagging replicas up to a configurable `shutdown-timeout`, by default 10 seconds, before shutting down.
+This provides a best effort minimizing the risk of data loss in a situation where no save points are configured and AOF is disabled.
+Before version 7.0, shutting down a heavily loaded master node in a diskless setup is more likely to result in data loss.
```

Comment:
```suggestion
Before version 7.0, shutting down a heavily loaded master node in a diskless setup was more likely to result in data loss.
```

Reply:
Right. I was thinking about users who continue to run old versions. For them it "is" still the case, but if we use "Before" we should also use "was". Another way to describe it would be "In versions prior to 7.0, ... is ....", but I like "before" because it's simpler language.